azure-pipelines.yml

```diff
@@ -0,0 +1,18 @@
+trigger:
+- master
```

Comment:
I like doing this on master, but surely the pytest tests don't exist there yet.....

Reply:
next time we merge they will :-)

Reply:
Should I remove it?

Reply:
Actually since this is only in the dev  branch it won't affect master until we check this into master.